### DISCUSSION TITLE
Installation Error - 'FileFinder' object has no attribute 'find_module'

### DISCUSSION THREAD

**Discussion by andreabnes** [Project-MONAI/MONAI#7705] [Q&A]
Hello Everyone,

I am trying to install MONAI 1.3.0 on a fresh Anaconda environment with "conda install -c conda-forge monai". The installation seems to proceed OK, but when I try to import monai from python3, I get this error:

(ai) andrea@ubuntu-2024-uefi:~$ python3
Python 3.12.3 | packaged by Anaconda, Inc. | (main, Apr 19 2024, 16:50:38) [GCC 11.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import monai
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/andrea/anaconda3/envs/ai/lib/python3.12/site-packages/monai/__init__.py", line 58, in <module>
    load_submodules(sys.modules[__name__], False, exclude_pattern=excludes)
  File "/home/andrea/anaconda3/envs/ai/lib/python3.12/site-packages/monai/utils/module.py", line 212, in load_submodules
    importer.find_module(name).load_module(name)  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^
AttributeError: 'FileFinder' object has no attribute 'find_module'

Has anyone come across the same issue?

The system is an Ubuntu 22.04 server, with CUDA 12.4 update 1, with Anaconda3-2024.02-1, and Pythorch 2.3.0

Any comment, suggestion is welcome.

Thank you!

Best Regards, Andrea

**Comment by andreabnes** [comment]
Hello All,

I have solved the issue by creating a new anaconda environment based on Python 3.11 (instead of 3.12)

This solution was prompted by this post https://github.com/Nuitka/Nuitka/issues/2369 

Best Regards,

Andreea

**Reply by KumoLiu** [reply]
The issue is fixed in this pr: https://github.com/Project-MONAI/MONAI/pull/7482
You can try latest version.
Thanks.

**Reply by andreabnes** [reply]
Thank you !
